File: CC/TrabalhoV/arquivosLua/micro05.lua
-- Listagem 17: Lê strings e caracteres
--[[ Função: Escrever um algoritmo que leia o nome e o sexo de 56 pessoas e informe o nome e se ela é homem ou mulher. No final informe o total de homens e mulheres --]]

h = 0
m = 0
for x=1,5,1
do
	print("Digite o nome: ")
	nome = io.read()
	print("H - Homem ou M - Mulher")
	sexo = io.read()
	if(sexo == "H") then h = h + 1
	elseif (sexo == "M") then m = m + 1
	else print("Sexo só pode ser H ou M!")
	end
end

print("Foram inseridos",h,"homens")
print("Foram inseridas",m,"mulheres")

File: CC/java/micro05.java

public class micro05{

    public static void main(String[] args){
        int x,m = 0, h = 0;
        String sexo, nome; 

        for(x = 0; x < 5; x++){
            System.out.println("Digite o nome: ");
            nome = System.console().readLine();

            System.out.println("H - homem ou M - mulher: ");
            sexo = System.console().readLine();

            switch(sexo){
                case "M":
                    m += 1;
                    break;
        
                case "H":
                    h += 1;
                    break;

                default:
                      System.out.println("Sexo só pode ser H ou M!"); 
            }
    
        } 

        System.out.println("Foram inseridos "+h+" homens.");
        System.out.println("Foram inseridos "+m+" mulheres.");
    }
}
